src/atomsToForm.ts

```diff
@@ -0,0 +1,54 @@
+import { atom } from 'jotai';
+import { SyncState } from './atomWithValidate';
+
+// Temporary type def
+type CollectionOfAtomValidate = {
+  [k: string]: any;
+};
+
+type FormArg = {
+  key: string;
+  value: unknown;
+};
+
+// TODO: add aync handling
+// TODO: add proper generics for infering the collection
+export const atomsToForm = (atoms: CollectionOfAtomValidate) => {
```

Comment:
Just one comment for now.
I'm not sure what naming convention we should go with.
For now, in the official repo, we use verb for creating atom from atom.
For example, `waitForAll`. Also, `splitAtom`,

Reply:
It's changeable anyway, we'll decide on it once the functionality's are done with